### ISSUE TITLE
Error in globalCallingHandlers when calling executeDqChecks

### ISSUE THREAD

**Issue by frasansa** [OHDSI/DataQualityDashboard#466]
I encountered an error while using the executeDqChecks function from the DataQualityDashboard package. When I called the function, the following error (with traceback) occurred:

```
Error in globalCallingHandlers(condition = conditionHandler) : 
should not be called with handlers on the stack
5.
globalCallingHandlers(condition = conditionHandler)
4.
registerDefaultHandlers()
3.
getLoggerSettings()
2.
ParallelLogger::registerLogger(logger = logger)
1.
DataQualityDashboard::executeDqChecks(connectionDetails, cdmDatabaseSchema = "main", 
resultsDatabaseSchema = "quality_checks", cdmSourceName = "Synthea synthetic health database", 
outputFolder = "qc_results", cdmVersion = "5.3", sqlOnly = TRUE, 
sqlOnlyIncrementalInsert = TRUE)
```

I am using:
R version 4.3.1 (2023-06-16 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19045)
DatabaseConnector_6.2.3

**Comment by katy-sadowski** [comment]
Hmm.  This seems like it might be a ParallelLogger issue - could you please share what version of ParallelLogger you have installed? (Also - assume you're on latest version of DQD?)

**Comment by frasansa** [comment]
Thank you, @katy-sadowski, for your message. I have installed ParallelLogger_3.2.0 and DataQualityDashboard_2.3.0. After further investigation, I found that the error only occurs when calling the function inside a quarto chunk, possibly due to a conflict with the error options configuration of Quarto in my R Studio environment. Fortunately, the error does not occur when running the .R script directly.
